pinot-controller/src/main/resources/app/pages/Query.tsx

```diff
@@ -635,9 +675,9 @@ const QueryPage = () => {
                       >
                         {
                           resultData.columns.length > 0 ? (
-                            <Typography variant='body2'>Partial results due to exceptions. Please toggle the switch to view details.</Typography>
```

Comment:
Now the message is printed by default, no need to switch to view details. Instead I've changed the text to say that stats may be partial